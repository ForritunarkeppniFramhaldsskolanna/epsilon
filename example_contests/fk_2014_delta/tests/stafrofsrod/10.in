11
fnegtkzwcrb
cqifeypdzbbhd
yqggzydlilhoarwwj
anmqozrpzuzih
qxtpzpyamxbcsnbnygq
unlxminehlonzaalv
ibhg
hrhrdbgdqglucmdgm
no
jm
rawbthgtzcxeyampk
