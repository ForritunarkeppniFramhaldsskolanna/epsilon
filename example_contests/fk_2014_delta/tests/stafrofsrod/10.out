anmqozrpzuzih
cqifeypdzbbhd
fnegtkzwcrb
hrhrdbgdqglucmdgm
ibhg
jm
no
qxtpzpyamxbcsnbnygq
rawbthgtzcxeyampk
unlxminehlonzaalv
yqggzydlilhoarwwj
